onnxruntime/test/providers/qnn/qnn_test_utils.cc

```diff
@@ -408,7 +414,7 @@ static BackendSupport GetHTPSupport(const onnxruntime::logging::Logger& logger)
   MockKernelLookup kernel_lookup;
   onnxruntime::GraphViewer graph_viewer(graph);
   std::unique_ptr<onnxruntime::IExecutionProvider> qnn_ep = QnnExecutionProviderWithOptions(
-      {{"backend_type", "htp"}, {"offload_graph_io_quantization", "0"}});
+      {{"backend_type", "htp"}, {"offload_graph_io_quantization", "0"}, {"skip_qnn_version_check", "1"}});
```

Comment:
why is `QNNTestEnvironment::GetInstance().skip_qnn_version_check()` not checked here?